k8s/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 // This file is part of MinIO Operator
-// Copyright (c) 2021 MinIO, Inc.
+// Copyright (c) 2022 MinIO, Inc.
```

Comment:
did you modify anything to change this? - if you change this please re-generate the code.

Reply:
I didn't change anything, but this way next time they regen code, it will have this header, I can regen all stubs, but I didn't want to make the main code of this PR hard to review :P @harshavardhana

Reply:
correct then lets do this change when you are going to regen assets, doing this halfway is not useful.  otherwise someone running regen locally will regen this part of the code everywhere.

Reply:
this change looks unrelated to this PR.

Reply:
removed it